### PR TITLE
[WIP] - Education/Employment Parsing

### PR DESCRIPTION
**Pull request by Safi-Ullah**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    for ent in entities:
+        entity = { 'org': ent.text }
+
+        head = get_root(ent.root.head)
+
+        nodes = deque(head.children)
+        nodes.appendleft(head)
+        while nodes:
+            node = nodes.popleft()
+            if entity_type == EntityType.Education and node.ent_type_ == 'DEGREE':
```
Comment (tayyab-razzaq) [inline utils.py]: we should store `DEGREE` in constant

File: utils.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                entity['degree'] = node.text
+            if entity_type == EntityType.Employment and node.ent_type_ == 'DESIGNATION':
```
Comment (tayyab-razzaq) [inline utils.py]: same as above

File: management/train_model.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                        annotations,  # batch of annotations
+                        drop=0,  # dropout - make it harder to memorise data
+                        losses=losses,
+                    )
+                print("Losses", losses)
+
+        if not model_dir.exists():
+            model_dir.mkdir()
+        nlp.to_disk(model_dir)
+        print("Saved model to", model_dir)
```
Comment (tayyab-razzaq) [inline management/train_model.py]: instead of printing to the console can we add a logging mechanism to file with log rotation

Comment (Safi-Ullah) [reply management/train_model.py]: What should be the max file size for log rotation?

Comment (tayyab-razzaq) [reply management/train_model.py]: 10MB

**Comment by Safi-Ullah** [comment]
> we should also add CI and add flake8 for linting with a docstring and import as plugin

Wouldn't it be better to do it in a separate PR?

**Comment by tayyab-razzaq** [comment]
> > we should also add CI and add flake8 for linting with a docstring and import as plugin
> 
> Wouldn't it be better to do it in a separate PR?

yes it will be done in separate PR

**Comment by tayyab-razzaq** [comment]
we should also address the opened issue related to license

**Pull request merged**